microsetta_admin/templates/scan.html

```diff
@@ -305,6 +307,39 @@ <h4>Scan History</h4>
                 </td>
             </tr>
             <!-- End collapsible email template fields -->
+            <tr>
+                <td>Sample Observations:</td>
+            <td>
+                <b>Tube </b>
```

Comment:
The categories should be dynamically rendered, just like the actual observations, as they will vary from project to project (including the possibility of no categories/observations for many projects).